src/ProjectTemplates/Microsoft.Extensions.AI.Templates/src/ChatWithCustomData/ChatWithCustomData.Web/Components/Pages/Chat/Chat.razor

```diff
@@ -18,7 +18,7 @@
 </div>
 
 @code {
-    private readonly string systemPrompt = @"
+    private const string SystemPrompt = @"
```

Comment:
Using raw strings would be a bit more modern, and, as a bonus, it'd remove a bunch of the whitespace that ends up in the string and reduce the token count :)

Reply:
Ah, just saw your razor comment in the other pr